tests/helpers/test_plates_helper.py

```diff
@@ -51,12 +56,13 @@ def test_get_samples_for_source_plate_successful(
                 "rna_id": "RNA-S-00001-00000009",
                 "lab_id": "MK",
                 "lh_sample_uuid": "c068cfea-02be-4aec-a50d-631abcd34573",
+                "created_at": timestamp,
+                "date_picked": "",
             },
         ]
 
-        # compare list of dicts
-        pairs = zip(actual, expected)
-        assert any(x != y for x, y in pairs)
+        for sample in actual:
+            assert sample in expected
```

Comment:
You can simplify this further to just `assert all([sample in expected for sample in actual])` if you wanted